src/App.tsx

```diff
@@ -5,18 +5,15 @@ import { ActivePageProvider } from './context/activePageContext'
 import theme from 'styles/createTheme'
 import { DialogProvider } from 'context/dialogContext'
 import routes from 'routes'
-import { Auth0ProviderWithNavigate } from 'provider/Auth0ProviderWithNavigate'
 
 const App = () => {
   const content = useRoutes(routes)
   return (
-    <Auth0ProviderWithNavigate>
-      <ThemeProvider theme={theme}>
-        <DialogProvider>
-          <ActivePageProvider>{content}</ActivePageProvider>
-        </DialogProvider>
-      </ThemeProvider>
-    </Auth0ProviderWithNavigate>
+    <ThemeProvider theme={theme}>
```

Comment:
Please leave `<Auth0ProviderWithNavigate>` and import for it as well. Just comment out these lines. It works on localhost, but requires additional configuration on Netlify.

Reply:
Done.